src/components/HomeContainer/Management/BuildManagement.module.css

```diff
@@ -0,0 +1,55 @@
+.container {
+  padding: 0;
+  margin: 0;
+  background-color: var(--dark-theme-background);
+  color: var(--primary-white);
+  padding-block: 64px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-inline: 24px;
+  background-image: url("../../../assets/bg-pattern-home-3.svg");
+  background-position-x: calc(100% + 100px);
+  background-position-y: top;
+  background-repeat: no-repeat;
+}
+
+.rectangle {
+  width: 50px;
+  height: 4px;
+  background-color: var(--coral-color);
+  margin-bottom: 32px;
+}
+.header {
+  max-width: calc(100% - 100px);
+  margin-bottom: 32px;
+}
+
+@media (min-width: 768px) {
+  .container {
+    padding: 98px;
+  }
+  .header {
+    width: 445px;
+  }
+}
+
+@media (min-width: 960px) {
+  .container {
+    flex-direction: row;
+  }
+  .rectangle {
+    background-position-y: bottom;
```

Comment:
`background-position-y: bottom;` should be on `.container`
`.rectangle` doesn't need any media-related rules and can be deleted